From 1c6a5e36a3af93642813e744639b4d88fcaec3e0 Mon Sep 17 00:00:00 2001
From: dann frazier <dann.frazier@chainguard.dev>
Date: Tue, 15 Sep 2026 11:33:31 -0600
Subject: [PATCH 01/18] Revert "tests: printenv: fix a false failure on
 GNU/Hurd"

This reverts commit e4a3653bc318f20419c08a840eb6d3bd1ab24885.
NEWS left untouched.
---
 tests/misc/printenv.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tests/misc/printenv.sh b/tests/misc/printenv.sh
index b8c3696c4..a6eab09f6 100755
--- a/tests/misc/printenv.sh
+++ b/tests/misc/printenv.sh
@@ -102,8 +102,7 @@ done
 for qs in literal shell c; do
   env -i PATH="$PATH" QUOTING_STYLE=$qs 'a b'='c d' \
     printenv >out-t 2>err || fail=1
-  grep -vE '^["'"'"']?'\
-'(__CF_USER_TEXT_ENCODING|QUOTING_STYLE|(LD_ORIGIN_)?PATH)["'"'"']?=' \
+  grep -vE '^["'"'"']?(__CF_USER_TEXT_ENCODING|QUOTING_STYLE|PATH)["'"'"']?=' \
     out-t >out || framework_failure_
   compare exp-noargs-$qs out || fail=1
   compare /dev/null err || fail=1
-- 
2.53.0

